codec_test.go

```diff
@@ -66,4 +66,13 @@ func TestCodec(t *testing.T) {
 
```

Comment:
Line` 13` could you add a test for `c.ID()`?

Reply:
sure